### ISSUE TITLE
Extraneous messages printed to console

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3296]
I'm not sure just when this happens or how to reproduce it, but just recently I've been getting occasional messages printed to the console like this:

```
divideAnySplitter 3 widget(s) in 2342761469920
divideAnySplitter 3 widget(s) in 2342761469920
```

The same one often gets printed more than once.
```
Leo 6.7.3-devel, devel branch, build 27df347781
2023-04-21 17:05:08 -0500
Python 3.11.3, PyQt version 6.4.3
Windows 10 AMD64 (build 10.0.19044) SP0
```

**Comment by edreamleo** [comment]
@tbpassin Thanks for this report. As you say, the messages are erroneous: the `NestedSplitter` class may add widgets.

Also, there seems to be no need for the warning messages about the `frac` arg being out of range. The code may as well set it to `0.5'.

**Comment by edreamleo** [comment]
Closed via PR #3297.
